Automattic/wp-calypso
# Automattic Components: Update Form Label component references to use package component

Related to https://github.com/Automattic/martech/issues/2028
Tracking issue https://github.com/Automattic/martech/issues/2023

We should wait for https://github.com/Automattic/wp-calypso/pull/80350 to sit in production for several days before merging this PR. It will allow us to see if there were any problems with our migration

## Proposed Changes

* Following up on https://github.com/Automattic/wp-calypso/pull/79256 and https://github.com/Automattic/wp-calypso/pull/80350, this repoints any references to `FormLabel` from Calypso components to `@automattic/components`
* I recognize this is a change that touches many files. We effectively made this change in the background with  https://github.com/Automattic/wp-calypso/pull/80350, so all form label components will have been running the new/relocated code already. This PR repoints the reference from the wrapper to the "real" code, so I don't expect this to cause any issues when https://github.com/Automattic/wp-calypso/pull/80350 has been in production for long enough

I will spot-checked several instances on the front end to ensure proper behavior.

## Testing Instructions



* Make sure tests pass.
* Do a spot-check of FormLabel instances to ensure they still look/work as expected. ( see https://github.com/Automattic/wp-calypso/pull/80350 for how to spot-check )

## Pre-merge Checklist
